app/importers/pure_journals_importer.rb

```diff
@@ -14,10 +14,19 @@ def call
         j.title = item['titles'].first['value']
         j.publisher = Publisher.find_by(pure_uuid: item['publisher']['uuid']) if item['publisher']
         j.save!
+      rescue StandardError => e
+        log_error(e, {
+                    journal_id: j&.id,
+                    item: item
+                  })
       end
       pbar.increment unless Rails.env.test?
+    rescue StandardError => e
+      log_error(e, {})
```

Comment:
Tests aren't hitting this line. I don't know enough to say whether that's important or not.

Reply:
There are a few of these testing misses, which is intentional because due to the way the code is structured it's nearly impossible to have an rspec mock raise an exception that will hit this. Given that this code is simply calling a thoroughly tested method, and since it would be really tricky to pull this off, I didn't think it was worth ending up with a very brittle test suite just to test a method called with no params